packages/decision-tree/src/components/StepBar/StepBar.tsx

```diff
@@ -15,13 +15,22 @@ export const StepBar: FC<StepBarProps> = ({ steps, activeStep }) => (
                 active={<Stepper.Step.Number />}
               />
             </Stepper.Step.Indicator>
-            <Stepper.Step.Separator />
+            <ProgressBar
+              width="100%"
+              height="2px"
+              min={0}
+              max={100}
+              value={activeStep > index ? 100 : activeStep === index ? 50 : 0}
```

Comment:
why fixed to 50 instead of the actual progress?

Reply:
Becuase it could jump up and down depending on the avoided questions due to the questionnaire logic, Hans corroborated the behaviour, just 50% between steps.